### DISCUSSION TITLE
TextBox Input

### DISCUSSION THREAD

**Discussion by thellebuyck** [vikramlearning/blazorbootstrap#511] [Q&A]
So, I have a Blazor Web App server project setup and been messing with layouts, sidebars and such.  Pretty cool added on.   But am I missing something here?  Is there not a basic TextBox form component that I can use to caputre inputs.  I see the Number and Autocomplete and the like but what about free form text?

**Comment by gvreddy04** [comment]
@thellebuyck You can use the default <InputText> control.
Link: https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.components.forms.inputtext?view=aspnetcore-6.0

**Comment by gvreddy04** [comment]
@thellebuyck 

Example:

```cshtml
<InputText class="form-control" @bind-Value="invoice.CustomerName" Placeholder="Enter Customer Name" />
```
